config/secrets.yml

```diff
@@ -19,9 +19,13 @@
 
 development:
   secret_key_base: 5fc0bc43d7cfb12126b2c9567f35d96d0b384eeb94f3014fb20b12214fb839db11166e7341e2c0acbf650c55d0944b4d05dbf8f47263204397fee769768d27b8
+  gthc_email: 'no-reply@gthc.io'
+  gthc_email_password: 'wDSg7J5Ak4ZJajkK'
 
 test:
   secret_key_base: 3cce65b14b6dd7f638ab3223387ef44e13034915a2e79656a84824a72eddb7492fa98d8818a6e85f1fd5ecced7afce6b4b94e8fe0adc76649d059ce85d27e759
+  gthc_email: 'no-reply@gthc.io'
```

Comment:
hey anesu can you please stop making this mistake

Reply:
once it something has been commited, it actually has technically has gone public and you would need to remove from the git history, but just for the sake of time, please stop making this mistake, it is getting annoying now. This is like the 5th time or something...